### PR TITLE
Fix TestDeleteWorkflowExecution_JustTerminatedWorkflow flaky test

### PR DESCRIPTION
**Pull request by alexshtin**
## What changed?

Fix `TestDeleteWorkflowExecution_JustTerminatedWorkflow` (former `Test_DeleteWorkflowExecution_RunningWithTerminate`) flaky test.

## Why?

Test fails from time to time w/o clear reason.

## How did you test it?

Existing tests.

### CODE REVIEW

File: tests/workflow_delete_execution.go [hunk 1]
```diff
@@ -34,42 +34,42 @@ import (
...
 const (
-	waitForPersistence = 10 * time.Second
+	// Internal task processing for DeleteExecutionTask checks if there is no pending CloseExecutionTask
+	// comparing ack levels of the last processed task and DeleteExecutionTask TaskID.
+	// Queue states/ack levels are updated with delay from "history.transferProcessorUpdateAckInterval"
+	// which default 30s, but it is overridden in tests to 1s (see overrideHistoryDynamicConfig in onebox.go).
+	// With few executions closed and deleted in parallel, it is hard to predict time needed for every DeleteExecutionTask
+	// to process. Set it to 20s here, as minimum sufficient interval. Increase it, if tests in this file are failing with
+	// "Condition never satisfied" error.
+	waitForTaskProcessing = 20 * time.Second
```
Comment (alexshtin) [inline tests/workflow_delete_execution.go:51]: This is a fix :-)

Comment (tdeebswihart) [reply tests/workflow_delete_execution.go:51]: Yikes :(

**Review by tdeebswihart — APPROVED** [review]

File: tests/workflow_delete_execution.go (hunk 1 above)
Comment (yycptt) [reply tests/workflow_delete_execution.go:51]: wow, it takes more than 10s to verify the task is completed even when UpdateAckInterval is 1s?

**Review by yycptt — APPROVED** [review]

File: tests/workflow_delete_execution.go (hunk 1 above)
Comment (alexshtin) [reply tests/workflow_delete_execution.go:51]: Yes :-(. Luckly, it is not a problem outside of the test.

**Pull request merged**